app/src/main/java/org/gateshipone/odyssey/fragments/SettingsFragment.java

```diff
@@ -180,6 +180,14 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
             getActivity().finish();
             startActivity(intent);
         }
+
+        if (key.equals(getString(R.string.pref_notification_private_key))) {
+            try {
+                boolean notificationPrivate = sharedPreferences.getBoolean(key, getResources().getBoolean(R.bool.pref_notification_private_default));
+                mServiceConnection.getPBS().notificationPrivateChanged(notificationPrivate);
```

Comment:
We had to change the connection handling and removed the mServiceConnection member. Please use the getPlaybackService method from the GenericActivity instead.